lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1225,8 +1229,17 @@ def __init__(
 
     def _on_lifecycle_event(self, _):
         # Upgrade event or other charm-level event
+        should_update = False
         for relation in self._charm.model.relations[self._relation_name]:
-            self._process_logging_relation_changed(relation)
+            # Don't accidentally flip a True result back.
+            should_update = should_update or self._process_logging_relation_changed(relation)
+        if should_update:
+            # We don't have an event, so build it up by hand
+            first_rel = self._charm.model.relations[self._relation_name][0]
```

Comment:
`first_rel` is just a dummy for building the event?
If it's not a dummy, why [0] is sufficient?
Add comment about it?